src/test/java/com/snack/news/service/AdminServiceTest.java

```diff
@@ -18,7 +19,8 @@
 import java.util.List;
 
 import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.Mockito.when;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'org.mockito.Mockito'](https://app.codacy.com/app/snack-news/Snack-BE/pullRequest?prid=4508329)